Infrastructure/PluginExecutor.cs

```diff
@@ -31,6 +33,11 @@ public PluginExecutor(IMessageSender messageSender, IDal dal, PluginsMenu plugin
             _pluginsMenu = pluginsMenu;
             _pluginsManager = pluginsManager;
         }
+        public string getPluginId(string user)
```

Comment:
1. The function name should be in 'pascal case'
2. Please give the function more meaningful name (Something as 'GetCurrentUserPluginId')

Reply:
OK, I changed as you requested